build/lib/test/test.js

```diff
@@ -402,7 +404,7 @@ async function runBuild(platform, target, deviceId, deployType, deviceFamily, sn
 		'--platform', platform,
 		'--target', target,
 		'--sdk', sdkVersion,
-		'--log-level', 'info'
+		'--log-level', logLevel
```

Comment:
not tested it yet but is the default `info` if you don't set it somewhere? Or is it undefined/empty and it needs to be set as a default value in the parameter list to `, logLevel = 'info')` ?

Reply:
Thanks @m1ga for reviewing.

The default value is set here:
https://github.com/tidev/titanium-sdk/blob/23b207b8d7a5832019d4cf8f79eb13afa5c79667/build/scons-test.js#L13

See also https://github.com/tj/commander.js?tab=readme-ov-file#default-option-value.